src/components/Comments.jsx

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from 'react';
+import '../App.css';
+import { getCommentsByArticleIdApi } from '../utils/api';
+
+const Comments =({singleArticleID}) => {
+  
+  const [comments, setComments] = useState([]);
+  
+
+  useEffect(() => {
+    if (singleArticleID) {
+    getCommentsByArticleIdApi(singleArticleID).then((CommentsFromArticleById) => {     
+      setComments(CommentsFromArticleById);    
+  });
+}
+},[]);
+  
+  return (
+<section>
+<h2 id="underlined">COMMENTS SECTION</h2>
+
+<ul className="">
+      {comments.map((comment) => {
+return (
+  <li className = "comment-box" key={comment.comment_id}>
+<p>{comment.author}</p>
+<p>{comment.body}</p>
+<p>{comment.created_at}</p>
```

Comment:
This date format isn't super readable at the moment, you could reformat this yourself, or have a library like Day.js do the hard work

Reply:
Ok will look at Day.js to sort it out.